src/components/Login/LoginOTPCard.tsx

```diff
@@ -141,7 +177,9 @@ export const LoginOTPCard: FunctionComponent<LoginOTPCard> = ({
           ) : (
             <SecondaryButton
               text="Resend"
-              onPress={resendOTP}
+              onPress={
+                lastResendWarningMessage === "" ? resendOTP : alertBeforeResend
+              }
```

Comment:
Why not let `LoginContainer` handle whether to show the resend warning message? Recall we want to show this on pressing submit after entering phone number too right?

Reply:
hmm the alert should not show after entering phone number for now, but yeah ill move it up in case this is supported in the future